tests/jballerina-unit-test/src/test/resources/test-src/functions/functions_with_included_record_parameters_negative.bal

```diff
@@ -258,3 +258,19 @@ function fn4(*Record4 record4) {
 function functionWithIncludedRecordParam18() {
     fn4(rec = {}, record4 = {}); // error
 }
+
+function fn(*Boo f) {
+
+}
+
+class MyClass {
+    function fn(*Boo f) {
+
+    }
+}
+
+function testInvokingFunctionWithUndefindeIncludedRecordParamUsingNamedArg() {
+    fn(f = {});
+    MyClass obj = new;
+    _ = obj.fn(f = {});
```

Comment:
Shall we add a comment here saying no errors will be logged for these lines, but we are testing the call path. For someone not familiar with the test this may look like unnecessary code.

Reply:
Function name contains those information right?

Reply:
I think it'll help to have a comment saying "No errors will be logged for the call expressions."

Reply:
Added in [b757e8b](https://github.com/ballerina-platform/ballerina-lang/pull/37914/commits/b757e8b3c3e703d5645e1ab0a098e0996261254e)